From 51eff361502cd679b3a5d736da2df3f40c8783f2 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Wed, 29 Jul 2020 20:10:24 +0200
Subject: [PATCH 3/3] Set MODULE_VERSION to 5.0.0

Without this the generated pkg-config files are invalid.

Qt5Contacts.pc gets generated but e.g. Qt5Versit.pc references
'Qt0Contacts' as dependency which can't be found.

As the libraries are also called libQt5* by the build system, let's set
MODULE_VERSION to 5.0.0.

Change-Id: I8f0bcb7cbdfef04e9c0a2416085832254342bc3d
---
 .qmake.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.qmake.conf b/.qmake.conf
index effd8983..a50b3c3b 100644
--- a/.qmake.conf
+++ b/.qmake.conf
@@ -2,4 +2,4 @@ load(qt_build_config)
 
 DEFINES += QT_NO_JAVA_STYLE_ITERATORS
 
-MODULE_VERSION = 0.0.0
+MODULE_VERSION = 5.0.0
-- 
2.27.0

